project/plugins.sbt

```diff
@@ -14,16 +14,29 @@
  * limitations under the License.
  */
 
-resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
+credentials += Credentials(Path.userHome / ".sbt" / ".credentials")
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers ++= Seq(
+  Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns),
+  Resolver.url("scoverage-bintray", url("https://dl.bintray.com/sksamuel/sbt-plugins/"))(Resolver.ivyStylePatterns)
+)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.13.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "0.8.4")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "0.10.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-settings" % "3.5.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.12")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "1.0.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-stamp" % "5.3.0")
```

Comment:
latest is `5.5.0` - worth checking the rest of the plugins to ensure all are at the latest